lib/tickle.rb

```diff
@@ -78,16 +79,16 @@ def bump(attr, amount=nil)
     when :hour then
       Time.local(self.year, self.month, self.day, self.hour + amount, self.min, self.sec)
     when :day then
-      Time.local(self.year, self.month, self.day + amount, self.hour, self.min, self.sec)
+      Time.local(self.year, self.month, self.day, self.hour, self.min, self.sec) + amount.days
```

Comment:
The problem with saying ```self.day + amount``` is that if the math creates an invalid date, an error is thrown.

I was able to test for this, and I fixed with activesupport.